### ISSUE TITLE
Hybrid UI demo page doesn't work properly on Windows Phone devices

### ISSUE THREAD

**Issue by TsvetinaIvanova** [telerik/kendo-ui-core#2713]
### Bug report

Kendo Hybrid UI demos don't work on Windows Phone devices.

### Reproduction of the problem
Open http://demos.telerik.com/kendo-ui/m/index in IE on Windows Phone 8.1 and 10 devices.

### Current behavior
On Windows Phone 8.1, the page doesn't scroll.
On Windows (Phone) 10, the page appears zoomed in and quite unresponsive.

* **Kendo UI version:** 2017.1.118
* **Browser:** Windows Phone 8.1 and 10

**Comment by TsvetinaIvanova** [comment]
The scrolling problem seems related to the fix of [#1830](https://github.com/telerik/kendo-ui-core/issues/1830).

It occurs for ListViews in native scrolling scenarios. If someone needs a quick fix, it is to overwrite the ListView touch-action with code like this, executed after the ListView is created (e.g. view afterShow event):
```
        $(".km-listview").css("touch-action", "pan-y");
        $(".km-listview").css("-ms-touch-action", "pan-y");
```

**Comment by Dimitar-Goshev** [comment]
Closing this issue as IE is not supported since R1 2023.
